pipelines/apDetectorVisitQualityCore.yaml

```diff
@@ -105,12 +105,20 @@ tasks:
       atools.interpolatedTemplatePsfSize.produce.plot.zAxisLabel: pixels
       python: |
         from lsst.analysis.tools.atools import DiffimSpatialMetricsHistPlot, DiffimSpatialMetricsInterpolatePlot
+  initialPviCore:
+    class: lsst.analysis.tools.tasks.CalexpSummaryAnalysisTask
+    config:
+      atools.calexpSummaryMetrics: CalexpSummaryMetrics
+      connections.data: initial_pvi.summaryStats
+      connections.outputName: initialPviSummary
```

Comment:
Add:
`connections.fakesType: parameters.fakesType`

Reply:
Unless I don't understand something about how analysis tools works (possible!), I don't think this is needed: Unlike the other Task inputs in this pipeline, the `initial_pvi` dataset type is upstream of all fakes data products.

Reply:
Ah, in that case you are right and it should not be needed.